tox.ini

```
@@ -0,0 +1,17 @@
+[tox]
+envlist = py35-django22{,-user_orgs}, py38-django22{,-user_orgs}
+
+[testenv]
+setenv =
+    DJANGO_SETTINGS_MODULE = test_settings
+    PYTHONPATH = {toxinidir}
+    TAHOE_SITES_USE_ORGS_MODELS = False
+    use_orgs: TAHOE_SITES_USE_ORGS_MODELS = True
```

`noorgs` explicit is better than implicit

```suggestion
    noorgs: TAHOE_SITES_USE_ORGS_MODELS = False
    withorgs: TAHOE_SITES_USE_ORGS_MODELS = True
```